starter-code/src/viking.js

```diff
@@ -1,11 +1,114 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(healthArg, strengthArg) {
+    this.health = healthArg;
+    this.strength = strengthArg;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(arg) {
+    this.health = this.health - arg;
```

Comment:
Rien d'obligatoire mais tu peux te servir de "-=" ici pour ne pas répéter this.health.